lib/configure.js

```diff
@@ -71,13 +71,6 @@ module.exports = function() {
   // without having to pass them down the chain.
   var slug, user, repo, username, password, otpCode, token, tempToken, tempTokenId;
 
-  // XXX Will users sometimes want to configure Travis to deploy changes to a
-  // non-"origin" repository?  For example, origin is mykmelez/test-app, but I
-  // want to configure Travis to deploy changes to mozilla/test-app, which has
-  // a different remote name (like upstream).  If so, then we'll need to prompt
-  // the user to choose the remote for which to configure deployment. For now,
-  // though, we assume they are configuring the origin remote.
-
```

Comment:
I just remembered the reason why this comment is all the way up here: the first thing this code does is set *slug* to the slug for the *origin* remote (using the *github-slug* module). And we use that in the initial message (`Configuring Travis to auto-deploy mozilla/oghliner to GitHub Pages…`) as well the token notes (`Oghliner token for mozilla/oghliner`, `temporary Oghliner token to get Travis token for mozilla/oghliner`) and, most importantly, when encrypting the GitHub token for use by Travis (which requires you to specify the slug of the repository for which the token can be used).

(We also use it to set a default value for the *username* field. We might want to add some smarts to guess the username better when there are both *origin* and *upstream* remotes, since the username is probably the one from the *origin* remote, even if the user is auto-deploying the *upstream* remote.)

So we should determine the repository for which we're configuring auto-deploy up here, before we have to use it. I wouldn't prompt before displaying the initial message, however, since that would be too abrupt, and it wouldn't give the user a chance to learn what the command will do first.

Thus, I'd remove the slug from the initial message and display something like this (basing this on the new language in #149):

```
Configuring Travis to auto-deploy to GitHub Pages.

Your repository has multiple remotes; please specify the one to auto-deploy.

Remote [default: upstream]: 

Ok, I'll configure Travis to auto-deploy the [REMOTE] remote ([SLUG FOR REMOTE]).
```

Or, if there's only one remote:

```
Configuring Travis to auto-deploy app to GitHub Pages.

Your repository has a single remote, [REMOTE] ([SLUG FOR REMOTE]).
That's what I'll configure Travis to auto-deploy.
```

(Wordsmithing welcome!)